app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SponsorshipsViewModel.java

```diff
@@ -0,0 +1,67 @@
+package io.github.droidkaigi.confsched2017.viewmodel;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+import com.annimon.stream.Collectors;
+import com.annimon.stream.Stream;
+
+import android.content.Context;
+import android.databinding.BaseObservable;
+import android.support.annotation.Nullable;
+
+import java.lang.reflect.Type;
+import java.util.List;
+
+import javax.inject.Inject;
+
+import io.github.droidkaigi.confsched2017.R;
+import io.github.droidkaigi.confsched2017.model.Sponsorship;
+import io.github.droidkaigi.confsched2017.util.AssetsUtil;
+import io.reactivex.Single;
+
+public final class SponsorshipsViewModel extends BaseObservable implements ViewModel {
+
+    private Context context;
+
+    @Inject
+    SponsorshipsViewModel(Context context) {
+        this.context = context;
+    }
+
+    @Override
+    public void destroy() {
+        // No-op
+    }
+
+    public Single<List<SponsorshipViewModel>> getSponsorships() {
+        return loadSponsors().map(sponsorships ->
+                Stream.of(sponsorships).map(SponsorshipViewModel::new).collect(Collectors.toList()));
+    }
+
+
+    private Single<List<Sponsorship>> loadSponsors() {
+        return Single.create(emitter -> {
+            try {
```

Comment:
This try catch is unnecessary.
See this PR 😃  https://github.com/DroidKaigi/conference-app-2017/pull/215